Update README.md (#125)
Evolution-App/iOS: README.md

# EVOlution - iOS

![Contributors](https://img.shields.io/github/contributors/Evolution-App/iOS?style=plastic)
![Forks](https://img.shields.io/github/forks/Evolution-App/iOS)
![Stars](https://img.shields.io/github/stars/Evolution-App/iOS)

> ![Licence](https://img.shields.io/github/license/Evolution-App/iOS)

Reviewer: Thanks for the badges!

> ![Issues](https://img.shields.io/github/issues/Evolution-App/iOS)

Reviewer: The Issues are not enabled in this repository. 

```suggestion
```

## Table of Contents

- 📋 [Description](#description)
- 📦 [How to Install/Run](#how-to-install/run)
- 💻 [Usage](#usage)
- 📝 [How to Contribute](#how-to-contribute)
- 🧑‍💻 [Author](#author)
- ✨ [Contributors](#contributors)
- ⚖️ [License](#license)

## Description

The initial goal of this project was to recreate the [Swift Evolution website](https://apple.github.io/swift-evolution) as an iOS app. 

Now, the goals have pivoted and we are shifting to make it more social for the next version. 

On our roadmap (you can read at our [GitHub Projects](https://github.com/Evolution-App/iOS/projects/2)), we have some notes for the future, which will cover stuff like: Follow Proposals starring or watching them, Follow Authors and/or Review Managers, List of most popular proposals, featured proposals and other ideas coming from the community.

![](images/screenshots_base.png)

## How to Install/Run

1. Fork and clone this repo to your local environment
2. Open the project file in Xcode

> 3. Set the target to am iPhone simulator or your iPhone device

Reviewer: ```suggestion
3. Select iPhone/iPad simulator or a real device to run the app
```

> 4. Click the 'Run' button on the top-left corner or use 'cmd + r' to run the code

Reviewer: ```suggestion
4. Click the 'Run' button on the top-left corner or use 'Command + R' to run the code
```

 
## Usage

To use this app, you can follow the installation steps above and choose the target to be your device or you can download the app from the [App Store](https://evoapp.io/s) for version 1.3. Use this app to see updates on the upcoming major and minor releases and proposals for changes to Swift on your mobile device.

## How to Contribute

> You need to create an issue and associate a pull request to this issue. Your pull request, needs to have some description on title about the issue that you are trying to solve. Please ensure that you follow the [Contribution Guidelines](.github/CONTRIBUTING.md). 

Reviewer: ```suggestion
Your pull request, needs to have some description on title about the problem you are solving. Please ensure that you follow the [Contribution Guidelines](.github/CONTRIBUTING.md). 
```
